pallets/funding/src/tests/1_application.rs

```diff
@@ -557,6 +557,44 @@ mod create_project_extrinsic {
 				);
 			});
 		}
+
+		#[test]
+		fn unaccepted_ct_decimals() {
```

Comment:
Please add accepted ct decimals test as well. Or add it to this test if you want.